packages/cloud_functions/cloud_functions/android/build.gradle

```diff
@@ -64,6 +64,7 @@ android {
     implementation platform("com.google.firebase:firebase-bom:${getRootProjectExtOrCoreProperty("FirebaseSDKVersion", firebaseCoreProject)}")
     implementation 'com.google.firebase:firebase-functions'
     implementation 'androidx.annotation:annotation:1.7.0'
+    implementation 'org.reactivestreams:reactive-streams:1.0.4'
```

Comment:
I'm not sure why we would need an external library to handle this? We managed to do all the other implementations without it AFAIK/

Reply:
Yes. The other ones provided a way to add listeners directly (eg. addOnConfigUpdatedListener) whereas this one doesn't. It returns a `Publisher` so this is the only way around it AFAIK.
Reference: https://firebase.google.com/docs/reference/kotlin/com/google/firebase/functions/HttpsCallableReference#summary